doc/tools/coqrst/coqdomain.py

```diff
@@ -99,6 +100,9 @@ def make_math_node(latex, docname, nowrap):
     node['number'] = None
     return node
 
+def myquote(tag):
+    return tag.replace(" ", "%20")
```

Comment:
I find it confusing to add a % sign in IDs, since they'll need further escaping when shown in a URL.  I think the usual approach is to use a - or a +.